### PR TITLE
MODULES-10104 - remove deploy to forge from travis

### PR DESCRIPTION
**Pull request by lionce**
bundle install --path .bundle/gems/
updated .sync file
pdk update

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-dist: trusty
```
Comment (glennsarti) [inline .travis.yml]: Shouldn't be changing this as part of this PR.

File: .sync.yml [hunk 2]
```diff
@@ -7,11 +7,10 @@
   unmanaged: true
 
 .travis.yml:
-  dist: trusty
+  deploy_to_forge:
```
Comment (glennsarti) [inline .sync.yml]: Shouldn't be changing this as part of this PR.

File: Gemfile [hunk 3]
```diff
@@ -1,5 +1,4 @@
 source ENV['GEM_SOURCE'] || 'https://rubygems.org'
-gem 'pdk', git: 'https://github.com/puppetlabs/pdk.git', branch: 'master'
```
Comment (glennsarti) [inline Gemfile]: Probably want to mention why the Gemfile is being changed in a PR about Travis.

File: metadata.json [hunk 4]
```diff
@@ -66,7 +66,7 @@
       "version_requirement": ">= 5.5.10 < 7.0.0"
     }
   ],
-  "pdk-version": "1.14.1",
+  "pdk-version": "1.15.0.pre (heads/master-0-gf5fdf1a)",
```
Comment (glennsarti) [inline metadata.json]: Shouldn't be using non-released versions of PDK on production modules.

Comment (lionce) [reply metadata.json]: all the changes outside sync file were done by pdk update

File: Gemfile (hunk 3 above)
Comment (lionce) [reply Gemfile]: this file was changed by pdk_update

Comment (lionce) [reply Gemfile]: you can check Gemfile for [facter_task](https://github.com/puppetlabs/puppetlabs-facter_task/blob/db743cdf2746d870bfc35403f656cd1b54bee5da/Gemfile#L2)

**Pull request merged**